libraries/AP_Scripting/examples/rover-set-turn-rate.lua

```diff
@@ -0,0 +1,58 @@
+-- This scirpt uses Rover's turn rate controller to make the vehicle move in circles of fixed radius
+
+-- Edit these variables
+local rad_xy_m = 0.5                -- circle radius in xy plane in m
+local target_speed_xy_mps = 0.5     -- target speed in m/s
+local rc_channel_switch = 7         -- switch this channel to "high" to get the script working
```

Comment:
It's not a blocker but we have aux functions for use by scripting now so you could use these instead to provide more flexibility to the user.